install/linux/usr/share/odemis/sim/sparc2-independent-ebic-sim.odm.yaml

```diff
@@ -0,0 +1,357 @@
+        limits: [-3, 3] # V
+    },
+}
+
+# CL PMT: composed of the control unit + the PMT output connected to the DAQ board
+
+# Must be connected on AI 1/AI GND
+"CL PMT": { # the raw detector of the PMT
+    # Internal child of SEM Scan Interface, so no class
+    role: null,
+    init: {
+        channel: 1, # 0-> sawtooth waves, 1-> square waves
+        limits: [3, 0], # V, with contrast reversed
+    },
+}
+
+"CL PMT control unit": {
+    class: pmtctrl.PMTControl,
+    role: null,
+    power_supplier: "Power Control Unit",
+    init: {
+        port: "/dev/fake", # for simulator
+        prot_time: 0.0002,  # s
+        prot_curr: 50.e-6,  # A
+    }
+}
+
+"CL Detector": {
+    class: pmtctrl.PMT,
+    role: cl-detector,
+    init: {
+        settle_time: 0.1, # s, time to wait before detector is ready
+    },
+    dependencies: {
+       "detector": "CL PMT",
+       "pmt-control": "CL PMT control unit",
+    },
+}
+
+# A detector which is independent from the DAQ board. Though in reality, it's
+# synchronized on the newPixel TTL signal of the DAQ board scanner.
+"EBIC": {
+    class: simsem.IndependentDetector,
+    role: ebic-detector,
+    init: {
+#        image: "",
+    },
+}
+
+# In reality, this is a Zyla, but you need libandor3-dev to simulate an AndorCam3
+# Depending exactly on the configuration, it might also be used for spectrometer
+"Camera": {
+    class: andorcam2.AndorCam2,
+    role: ccd,
+    power_supplier: "Power Control Unit",
+    init: {
+       device: "fake",
+       image: "sparc-ar-mirror-align.h5", # only for simulator
+    },
+}
+
+"Spectrometer Vis-NIR": {
+    class: spectrometer.CompositedSpectrometer,
+    role: spectrometer,
+    dependencies: {detector: "Camera", spectrograph: "Spectrograph"},
+    init: {
+        transp: [1, -2], # only applied to the spectrometer data (not raw CCD)
+    },
+}
+
+"Spectrograph": {
+    class: andorshrk.Shamrock,
+    role: spectrograph,
+    power_supplier: "Power Control Unit",
+    init: {
+        device: "fake",
+        slits: {1: "slit-in", # INPUT_SIDE
+        },
+        bands: { # The filters in the filter-wheel
+            1: [320.e-9, 400.e-9], # pos -> m,m
+            2: [500.e-9, 550.e-9],
+            3: [400.e-9, 450.e-9],
+            4: "polariser",
+            6: "pass-through"
+        },
+        fstepsize: 10.9e-6, # m, size of one focus step (very roughly)
+        rng: {"focus": [10.9e-6, 4.9e-3]},  # the hardware has an issue, and cannot go further than 4.9mm (instead of the standard 6.5 mm)
+    },
+    affects: ["Camera", "Spectrometer Vis-NIR"],
+}
+
+"Spectrograph focus": {
+    class: actuator.MultiplexActuator,
+    role: "focus",
+    dependencies: {"z": "Spectrograph"},
+    init: {
+        axes_map: {"z": "focus"},
+    },
+    affects: ["Camera", "Spectrometer Vis-NIR"],
+}
+
+# Provide the mirror (aka port selector) of the spectrograph as a separate component
+"Spec CCD Flipper": {
```

Comment:
The Odemis optical path manager doesn't care. However, in reality, typically the spectrograph always has a flipper, and it's important to place it to the right position, because the flipper position is "random" at start-up.

I'd rather keep the simulator as similar as the real hardware, so keep this component.